docs/sources/administration/plugin-management/index.md

```diff
@@ -33,19 +37,19 @@ Use panel plugins when you want to:
 
 ## Data source plugins
 
-Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource).
+Data source plugins add support for new databases, such as [Google BigQuery](/grafana/plugins/grafana-bigquery-datasource), [Zendesk](https://grafana.com/grafana/plugins/gabrielthomasjacobs-zendesk-datasource/) and [NetData](https://grafana.com/grafana/plugins/netdatacloud-netdata-datasource/) Data source plugins.
```

Comment:
Required change: Zendesk and NetData aren't what I would think of as databases, and bookending the start and end with "Data source plugins" feels strange to me.

Reply:
The reason I mentioned them is that I want to share the combination of Data sources which are owned by Grafana (Zabbix), Community (Zendesk) and Partner (Netdata). 

If you suggest that it may not be the required then I can simply undo my changes here.